src/services/HttpService.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export default class HttpService {
+  constructor() {
+    this.client = axios.create({
+      baseURL: "http://localhost:8000/api",
+    });
+    this.client.interceptors.request.use(function (req) {
+      const token = localStorage.getItem("token");
```

Comment:
Ovo bi izmenila da se postavlja prilikom Logina a ne prilikom svakog requesta, ali to mozemo detaljnije kada budes radila Login.